src/Sentry/Sentry.csproj

```diff
@@ -8,6 +8,16 @@
     <Description>Official SDK for Sentry - Open-source error tracking that helps developers monitor and fix crashes in real time.</Description>
     <NoWarn Condition="$(TargetFramework) == 'netstandard2.0'">$(NoWarn);RS0017</NoWarn>
     <DefineConstants>$(AdditionalConstants)</DefineConstants>
+    <Configurations>Debug;Release;Release_Unity;Debug_Unity</Configurations>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="$(Configuration) == 'Release_Unity'">
```

Comment:
I believe it's best we add an additional Property instead so we can continue to use Configurion=Release
The reasoning is that there (?) properties and behavior tied to Configuration `Release` and we might be missing those due to introducing a new configuration. 

Can we just pass an additional properly: `dotnet msbuild /p:Unity=true` ?

Reply:
but that wont work when building from and ide